### ISSUE TITLE
[FEATURE] Prevent restock every 24 hours. 

### ISSUE THREAD

**Issue by StarWeizz** [ChimpGamer/UltimateMobCoins#4]
Hello,

I would like to request a feature that prevents automatic restocking every 24 hours. I want my shop to have a restock time of 48 hours, but currently, it restocks automatically every 24 hours.

Thanks.

**Comment by ChimpGamer** [comment]
Did you set the refresh_time of the shop to 48 hours? By default it is 24 hours but can be adjusted.

**Comment by StarWeizz** [comment]
Of course, but It resets stock every time I restart the server. And same if there is a crash of the server. Cf: https://discord.com/channels/222070253172031500/1263113707215650866

**Comment by ChimpGamer** [comment]
Ah so you mean that the data should persist over restarts and crashes. That was not mentioned in the issue.

**Comment by StarWeizz** [comment]
Exactly!

**Comment by ChimpGamer** [comment]
I've added this feature in 1.4.0-SNAPSHOT as experimental feature. You can download it here https://github.com/ChimpGamer/UltimateMobCoins/actions/runs/13837697663

**Comment by ChimpGamer** [comment]
Released 1.4.0 so I'm closing this issue now.
